cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -252,6 +253,183 @@ describe('Dashboards', () => {
     )
   })
 
+  describe('cloning', () => {
+    const localDashName = 'Happy Dashboard'
+    const labelName = 'bark'
+    const secondLabelName = 'prrr'
+    beforeEach(() => {
+      cy.writeLPDataFromFile({
+        filename: 'data/wumpus01.lp',
+        offset: '20m',
+        stagger: '1m',
+      }).should('equal', 'success')
+
+      return cy.get<Organization>('@org').then(({id}: Organization) =>
+        cy.createDashboard(id, localDashName).then(({body}) =>
+          cy.createAndAddLabel('dashboards', id, body.id, labelName).then(() =>
+            cy
+              .createLabel(secondLabelName, id, {
+                description: 'I hate mieces',
+                color: '#00ff44',
+              })
+              .then(() => {
+                cy.createCell(body.id).then(cell1Resp =>
```

Comment:
be carful of handling async tasks. Using the `{` here means that cypress is expecting a declared `return` statement. To avoid the before each ending prematurely, I would recommending removing the `{` and just using `=>` instead.

Reply:
curly braces removed